ui/app/src/modules/System/Sites/Edit/EditSiteDialog.tsx

```diff
@@ -144,13 +160,12 @@ function EditSiteDialog(props: EditSiteDialogProps) {
   };
 
   return (
-    <Dialog
-      open={open}
-      onClose={onClosed}
-      aria-labelledby="editSiteDialogTitle"
-      fullWidth
-      maxWidth="sm"
-    >
+    <>
+      <DialogHeader
+        id="editSiteDialogTitle"
+        title={<FormattedMessage id="editSiteDialog.title" defaultMessage="Edit Site" />}
+        onDismiss={onClose}
+      />
```

Comment:
What were these changes for?

Reply:
Moved to the original component and handled error using ErrorDialog